ocaml/xapi/quicktest_cbt.ml

```diff
@@ -123,7 +123,7 @@ let vdi_data_destroy_test ~session_id ~vDI =
       ~msg:"VDI.data_destroy failed to update VDI.type";
     assert_cbt_status true ~session_id ~test ~vDI:snapshot
       ~msg:"VDI snapshot cbt_enabled field erroneously set to false";
-    test_vdi_update ~session_id ~test snapshot;
+  (*  test_vdi_update ~session_id ~test snapshot; *)
```

Comment:
I think we could also mention here that it's blocked on CA-273981, that VDI.update currently does not work on a cbt_metadata VDI. This way we'll know that we can uncomment/remove this test when the issue is fixed/resolved.